packages/components/tooltip/src/TooltipTrigger.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useTooltipContext } from './TooltipContext';
+import { useMergeRefs } from '@floating-ui/react';
+
+export const TooltipTrigger = React.forwardRef<
+  HTMLElement,
+  React.HTMLProps<HTMLElement> & { tooltipId: string }
+>(function TooltipTrigger({ children, tooltipId, ...otherProps }, propRef) {
+  const context = useTooltipContext();
+
+  const childRef: React.Ref<HTMLElement> | undefined = (
+    children as unknown as { ref?: React.Ref<HTMLElement> }
+  ).ref;
+  const baseRef = useMergeRefs([propRef, context.refs.setReference]);
+  const ref = useMergeRefs([childRef, baseRef]);
+
+  if (React.isValidElement(children)) {
```

Comment:
Should we log an error at least in dev mode rather than silently hiding the element?